### PR TITLE
(#3986) - Split out android tests more

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -58,8 +58,8 @@ env:
   - CLIENT="saucelabs:internet explorer:10:Windows 8" COMMAND=test
 
   # split up the android tests as it goes over time
-  - GREP=test.replication.js CLIENT="saucelabs:Android:5.1:Linux" COMMAND=test
-  - GREP=test.replication.js INVERT=true CLIENT="saucelabs:Android:5.1:Linux" COMMAND=test
+  - GREP=test.replication.js,test.attachments.js CLIENT="saucelabs:Android:5.1:Linux" COMMAND=test
+  - GREP=test.replication.js,test.attachments.js INVERT=true CLIENT="saucelabs:Android:5.1:Linux" COMMAND=test
```
Comment (nolanlawson) [inline .travis.yml]: whoa, I didn't know you could put a comma in there. that is rad

**Comment by nolanlawson** [comment]
+1 when green, should help with android flakiness

File: .travis.yml (hunk 1 above)
Comment (daleharvey) [inline .travis.yml]: Heh I was sure I had something like this working, but it isnt now

**Comment by daleharvey** [comment]
ah, found the issue, | not ,

**Comment by daleharvey** [comment]
Not sure this is working correctly at all, seeing the basics tests run when we are grepping attachments + replication

**Comment by nolanlawson** [comment]
If it isn't, let's just add a string like "suite2" to the attachments and replication tests.

**Comment by daleharvey** [comment]
Dumb mistake, need to quote special characters that are gonna be run through bash

**Comment by daleharvey** [comment]
Ugh, that was a but now its fixed, there must be a bug in webview passing in parameters, the url we generate is working fine in the browser

**Comment by daleharvey** [comment]
Going with your suggestion @nolanlawson, debugging the param issue into webview didnt seem worth it

**Comment by daleharvey** [comment]
No restarts :) landed - https://github.com/pouchdb/pouchdb/commit/1d98f4b8283beebff615abaad1ce1d7174a18245

**Pull request closed without merging**